lib/imports/api/authority/server/authority.js

```diff
@@ -139,7 +141,13 @@ export class Authority {
     });
     // notify listening clients that new steps have arrived
     console.log('Authority notifying clients of new steps');
-    this.streamerServer.emit('authorityNewSteps');
+    const newStepsEventName = generateEventName({
```

Comment:
generate event names unique for this docId so that client only subscribes to this single doc